src/widgets/NewExpiryDateInput.js

```diff
@@ -29,45 +35,44 @@ import { parseExpiryDate, formatExpiryDate } from '../utilities';
  * @param {bool} disabled If `true` will render a plain Cell element with no interaction
  * @param {bool} isFocused If `false` will TouchableOpacity that dispatches a focusAction
  *                         when pressed. When `true` will render a TextInput with focus
- * @param {func} editAction Action creator for handling editing of this cell.
- *                          `(newValue, rowKey, columnKey) => {...}`
- * @param {func} focusAction Action creator for handling focusing of this cell.
- *                          `(rowKey, columnKey) => {...}`
- * @param {func} focusNextAction Action creator for handling focusing of this cell.
- *                          `(rowKey, columnKey) => {...}`
+
  * @param {func}  dispatch Reducer dispatch callback for handling actions
  * @param {Number}  width Optional flex property to inject into styles.
  * @param {Bool}  isLastCell Indicator for if this cell is the last
  *                                   in a row. Removing the borderRight if true.
  * @param {String}  placeholder String to display when the cell is empty.
+ * @param {func} editAction Action creator for handling editing of this cell.
+ *                          `(newValue, rowKey, columnKey) => {...}`
  *
  */
 
-const {
-  expiryBatchTextView,
-  expiryBatchView,
-  expiryBatchText,
-  expiryBatchPlaceholderText,
-} = newDataTableStyles;
+const { expiryBatchView, expiryBatchText, expiryBatchPlaceholderText } = newDataTableStyles;
 
 export const NewExpiryDateInput = React.memo(
   ({
     value,
     rowKey,
     columnKey,
     isDisabled,
-    isFocused,
+    placeholderColour,
     editAction,
-    focusAction,
-    focusNextAction,
     dispatch,
     isLastCell,
     width,
     debug,
     placeholder,
+    rowIndex,
   }) => {
     if (debug) console.log(`- ExpiryTextInputCell: ${value}`);
 
+    const { focusNextCell, getRefIndex, getCellRef } = React.useContext(RefContext);
+
+    console.log('#################################');
```

Comment:
is this debug code? (i.e. behind the `debug` flag) or temp code (i.e. shouldn't be here)

Reply:
38a9f10ddde43bf9e87908dcd76be4d3d6e04179